content/sensu-go/6.0/release-notes.md

```diff
@@ -66,6 +67,66 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
+- ([Commercial feature][162]) Docker-only Sensu now binds to the hostname of containers instead of `localhost`. Docker images now set their own default values for environment variables `SENSU_AGENT_API_URL`, `SENSU_BACKEND_API_URL`, `SENSU_BACKEND_ETCD_INITIAL_CLUSTER`, `SENSU_BACKEND_ETCD_ADVERTISE_CLUSTER`, `SENSU_BACKEND_ETCD_INITIAL_ADVERTISE_PEER_URLS`, `SENSU_BACKEND_ETCD_LISTEN_CLIENT_URLS`, and `ETCD_LISTEN_PEER_URLS`.
+- ([Commercial feature][162]) Added Linux packages for 386; armv5, armv6, and armv7; MIPS hard float, MIPS LE hard float, and MIPS 64 LE hard float; ppc64le; and s390x architectures.
+See the [supported platforms][165] page for a complete list of Sensu’s supported platforms.
+- Added [Sensu query expression][168] `sensu.CheckDependencies`.
+- Added [binary-only distributions][164] for FreeBSD `armv5`, `armv6`, and `armv7` and Linux `ppc64le` and `s390x`.
+- Added the `is_silenced` Boolean attribute to the event.Check object to indicate whether the event was silenced at the time it was processed.
+
+**IMPROVEMENTS:**
+
+- ([Commercial feature][162]) Added support for the `memberOf` attribute in Active Directory (AD).
+- ([Commercial feature][162]) Added more descriptive information for errors in the federated web UI.
+- The `dead` and `handleUpdate` methods in keepalived now use `EntityConfig` and `EntityState` respectively.
+- The `dead()` and `createProxyEntity()` methods in eventd now use `corev3.EntityConfig` and `corev3.EntityState`.
+- Agent entity updates now ignore state-related fields.
+- You can now manage Sensu agent configuration via the HTTP API.
+- For sysvinit services, Sensu now passes users' secondary groups (i.e. groups other than the Sensu user group) to `chroot`, which gives the Sensu agent and backend access to the file access writes that are granted to the secondary groups.
+- Output of `sensuctl asset add` now includes help for using the runtime asset.
+- For role bindings and cluster role bindings, [`subjects.name`][166] values can now include unicode characters, and [`roleRef.type`][167] and [`subjects.type`][166] values are now automatically capitalized.
+- Improved logging for the agent websocket connection.
+- Improved the wording of the secret provider error message.
+- Fewer keys in etcd are now stored for agents.
+- Keepalive and round robin scheduling leases are now dealt with more efficiently.
+- Upgraded Go version from 1.13.7 to 1.13.15.
+- Upgraded etcd version from 3.3.17 to 3.3.22.
+
+**FIXES:**
+
+- ([Commercial feature][162]) Label selectors now work as expected with multiple requirements for events.
+- ([Commercial feature][162]) Fixed an issue that prevented broken secrets providers from surfacing their errors.
+- ([Commercial feature][162]) Fixed a bug for PostgreSQL datastores that could prevent GraphQL from retrieving all pages when fetching events in a namespace with more than 1000 total events, resulting in an unexpected error.
```

Comment:
```suggestion
- ([Commercial feature][162]) Fixed a bug for PostgreSQL datastores that could prevent GraphQL from retrieving all pages when fetching events in a namespace with more than 1000 total events, resulting in an unexpected error.
- ([Commercial feature][162]) Fixed a bug that could cause the backend to panic in case of PostgreSQL errors.
```